src/components/IncreaseVotes.jsx

```diff
@@ -0,0 +1,19 @@
+import { updateVotes } from "../utils/api"
+
+const IncreaseVotes = ({setDispalyVotes, article_id}) => {
+    const handleIncrease = () => {
+        setDispalyVotes((currVotes) => {
+            return currVotes = currVotes + 1
+        })
```

Comment:
Do you want users to be able to vote in either direction as many times they want or just once? Up to you, though I'd argue it makes sense to limit how many votes a user can give. If you did we'd need some extra state to track if the user has already voted